tests/bud/buildkit-mount-from/Dockerfilebindfrom

```diff
@@ -0,0 +1,4 @@
+FROM alpine
+RUN mkdir /test
```

Comment:
Is this supposed to be required before it can be used as a mount point?

Reply:
Removed in latest commit from this file. No not necessary i only added it so i could test if directory is empty without mount.